mmv1/templates/terraform/examples/tpu_v2_queued_resource_basic.tf.tmpl

```diff
@@ -0,0 +1,19 @@
+resource "google_tpu_v2_queued_resource" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+
+  name    = "{{index $.Vars "qr_name"}}"
+  zone    = "us-central1-c"
+  project = "{{index $.TestEnvVars "project_id"}}"
```

Comment:
Currently the generated test and documentation from this example leaves this field as an empty string, as there isn't a project_id var. Using `project` fixes this.

```suggestion
  project = "{{index $.TestEnvVars "project"}}"
```

Reply:
Updated to `project`